client/my-sites/email/email-providers-stacked-comparison/provider-cards/professional-email-card.tsx

```diff
@@ -156,7 +156,11 @@ const ProfessionalEmailCard = ( {
 
 	professionalEmail.onExpandedChange = onExpandedChange;
 	professionalEmail.priceBadge = (
-		<ProfessionalEmailPrice domain={ domain } intervalLength={ intervalLength } />
+		<ProfessionalEmailPrice
+			domain={ domain }
+			intervalLength={ intervalLength }
+			isDomainInCart={ isDomainInCart }
+		/>
```

Comment:
```suggestion
		<ProfessionalEmailPrice {...{ domain, intervalLength, isDomainInCart } } />
```

Nit: We could save characters by just passing all props with the same name transparently ...